src/covid_model_seiir_pipeline/pipeline/forecasting/model/containers.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+from typing import Dict, List, Union
+
+import pandas as pd
+
+from covid_model_seiir_pipeline.lib import (
+    utilities,
+)
+
+# This is just exposing these containers from this namespace so we're not
+# importing from the regression stage everywhere.
+from covid_model_seiir_pipeline.pipeline.regression.model.containers import (
+    HospitalFatalityRatioData,
+    HospitalCensusData,
+    HospitalMetrics,
+    HospitalCorrectionFactors,
+)
+
+
+@dataclass
```

Comment:
Didn't you remove a lot of these dataclass objects in the previous PR? Why add these?